Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -112,6 +119,21 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
     cell.contentView.alpha = 0.5
     #endif
   }
+    
+  func startDetermineLocation() {
+    if CLLocationManager.locationServicesEnabled() {
+      self.locationManager.startUpdatingLocation()
+    }
+  }
+    
+  func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+    let userLocation:CLLocation = locations[0] as CLLocation
```

Comment:
Can this array be empty?

Reply:
`locations` is non-optional, `locations[0] != nil` will always be `true`.